src/main/java/github/weichware10/toolbox/zoommaps/Zoombildschirm.java

```diff
@@ -0,0 +1,62 @@
+package github.weichware10.toolbox.zoommaps;
+
+import github.weichware10.toolbox.gui.Endbildschirm;
+import github.weichware10.util.config.ConfigClient;
+import javafx.event.ActionEvent;
+import javafx.event.EventHandler;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+/**
+ * Kümmert sich um das Design des Tests Zoommaps.
+ */
+public class ZoomBildschirm {
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.OuterTypeFilenameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>The name of the outer type and the file do not match.